src/sauce.config.js

```diff
@@ -47,6 +47,10 @@ const overrides = {
   testIgnore: process.env.TEST_IGNORE,
 };
 
+if (!userConfig.workers) {
```

Comment:
Hm. So that means that the user is never able to override the number of workers that's specified in the playwright config via saucectl. Is that intentional?

Reply:
Yep. I think this is a priority issue. IMO the playwright config for the project has higher priority so it cannot be overridden here. Any convention or suggestion here? 🤔

Reply:
Did an experiment. If `workers` is set in both cli and playwright config, playwright picks the cli one. Just remove the overridden settings.